bundles/test/org.eclipse.smarthome.magic/src/main/java/org/eclipse/smarthome/magic/binding/handler/MagicConfigurableThingHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.smarthome.magic.binding.handler;
+
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+
+public class MagicConfigurableThingHandler extends BaseThingHandler {
+
+    public MagicConfigurableThingHandler(@NonNull Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(@NonNull ChannelUID channelUID, Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    @Override
+    public void handleConfigurationUpdate(Map<String, Object> configurationParameters) {
```

Comment:
What's the reason to implement that function if only the super method is called?

Reply:
I used it to set a break point there while debugging #1477 :) Since the magic bundle is our "testing bundle", maybe its useful for others too.

Reply:
Ah, okay for me